src/model/exceptions/active-part-exception.ts

```diff
@@ -0,0 +1,8 @@
+import { ErrorCode } from '../enums/error-code'
+import { Exception } from './exception'
+
+export class ActivePartException extends Exception {
```

Comment:
Suggestion: We could call it something like `AlreadyOnAirException`. Then the exception is a bit more reuseable and we don't risk as much having exceptions only use for a single specific things

Reply:
Do you want `ActivePartException` and `ActiveRundownException` merged together to `AlreadyOnAirException`?

Reply:
Initial thoughs: no. A Rundown being active doesn't necessarily means that it is on Air. You can have an active Rundown that has no Parts on Air (when we activate the Rundown before doing a Take)

Reply:
I have changed it to `OnAirException`. (ActiveRundown is still kept)